e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @onboarding @smoke
+
+describe('Onboarding', () => {
+    let testUser;
+    let otherUser;
+    let testTeam;
+
+    // let config;
+
+    before(() => {
+        // cy.apiGetConfig().then((_data) => {
+        cy.apiGetConfig().then(() => {
+            // ({config} = data);
+        });
+
+        // # Set Support Email setting
+        // const newSettings = {
+        //     SupportSettings: {
+        //         SupportEmail: 'feedback@mattermost.com',
+        //     },
+        // };
+        // cy.apiUpdateConfig(newSettings);
+
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            cy.apiCreateUser({bypassTutorial: false}).then(({user: user2}) => {
+                otherUser = user2;
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id);
+            });
+
+            cy.apiCreateUser({bypassTutorial: false}).then(({user: user1}) => {
+                testUser = user1;
+                cy.apiAddUserToTeam(testTeam.id, testUser.id);
+
+                cy.apiLogin(testUser);
+                cy.visit(`/${testTeam.name}/channels/town-square`);
+            });
+        });
+    });
+
+    it('Takes the user through the steps of using the app', () => {
```

Comment:
@neallred I wonder if there's a follow up PR on this spec file. If yes, then I'd suggest to coordinate with your QA so that a proper test criteria is written and saved into Zephyr. Then you may reference the Zephyr test case ID when writing Cypress script. Thanks and let me know if you have any question.